packages/sdk/src/DeterministicDeployer.ts

```diff
@@ -69,7 +69,12 @@ export class DeterministicDeployer {
       })
     }
     await this.provider.send('eth_sendRawTransaction', [DeterministicDeployer.deploymentTransaction])
-    if (!await this.isContractDeployed(DeterministicDeployer.proxyAddress)) {
+    var tryCount = 5
```

Comment:
can you explain on what networ it doesn't work?
this code deploys the standard "deterministic deployer", which afaik is deployed on all testnets and most mainnets.
it runs cleanly on local geth or hardhat.
anyway, if needed it is better to use`wait()` to wait for confirmation, rather than poll on contract size. (added here only as a "sanity" that the contract address and deployment code are as expected.